UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareServiceCache.cs

```diff
@@ -121,7 +124,17 @@ private Task<BatchDetail> CheckIfCacheProductsExistsInBlob(string exchangeSetRoo
             {
                 var bundleLocation = item.Bundle.FirstOrDefault().Location.Split(";");
                 exchangeSetRootPath = string.Format(exchangeSetRootPath, bundleLocation[0].Substring(1, 1), bundleLocation[1]);
-                downloadPath = Path.Combine(exchangeSetRootPath, item.ProductName.Substring(0, StringLength), item.ProductName, item.EditionNumber.Value.ToString());
+                if (aioConfiguration.IsAioEnabled)
+                {
+                    List<string> aioCells = !string.IsNullOrEmpty(aioConfiguration.AioCells) ? new(aioConfiguration.AioCells.Split(',')) : new List<string>();
+
+                    if (!aioCells.Contains(item.ProductName))
```

Comment:
Missing braces

Reply:
Resolved. Added braces. Not added braces previously as one liner code was there.